src/scripts/main.js

```diff
@@ -1,3 +1,341 @@
 'use strict';
 
-// write your code here
+const everySquare = Array.from(document.querySelectorAll('.field-cell'));
+const everyRow = Array.from(document.querySelectorAll('.field-row'));
+const mainButton = document.querySelector('.start');
+const gameScore = document.querySelector('.game-score');
+let canMoveInAnyDirection = [];
+let score = 0;
+
+function appendNumber() {
+  const emptySquares = everySquare.filter(square => square.innerHTML === '');
+
+  if (emptySquares.length === 0) {
+    return;
+  }
+
+  const index = Math.floor(Math.random() * emptySquares.length);
+  const randomSquare = emptySquares[index];
+  const value = Math.random() > 0.9 ? 4 : 2;
```

Comment:
'0.9', '4', '2' are 'magic number', so make sense to move these to the constants